src/Event/Dto/RemoveEventDto.php

```diff
@@ -20,10 +20,7 @@ public function __construct(object $source, $id)
         $this->id = $id;
     }
 
-    /**
-     * @return mixed
-     */
-    public function getId()
+    public function getId(): mixed
```

Comment:
This will break with PHP 7.x

Reply:
Ah alright, maybe include in the documentation somewhere that the php-cs-fixer should be ran with PHP 7.4? I ran it with PHP 8.1, which introduced these changes 😅

Reply:
Ah, so it is in fact ran with PHP 8.1, but the `phpdoc_to_return_type` option is set to `false` in the workflow. @DamienHarper Is there a reason as to why this is set to `false` in the workflow, but to `true` in the phpcs config file?

Reply:
Well it should be compatible with PHP 7.4 so PHP-CS-Fixer should be run with 7.4. 
In reality it can be run with 8.x but mostly should not alter mixed return type.